CommunityFridgeMapApi/tests/unit/test_fridge_report.py

```diff
@@ -9,130 +9,171 @@ def __init__(self):
     def put_item(self, TableName=None, Item=None, ConditionExpression=None):
         pass
 
+    def get_item(self, TableName=None, Key=None):
+        return {"Item": {"json_data": {"S": '{"id": {"S": "test"}}'}}}
 
-class FrdgeReportTest(unittest.TestCase):
-    def test_required_fields(self):
-        fridge_report = FridgeReport(
-            fridge_report={
-                "fridge_id": "test",
-                "status": "working",
-                "notes": "all good out here",
-            },
-            db_client=None,
-        )
-        success, field = fridge_report.has_required_fields()
-        self.assertFalse(success)
-        self.assertEqual(field, "fridge_percentage")
+    def update_item(
+        self,
+        TableName=None,
+        Key=None,
+        ExpressionAttributeNames=None,
+        ExpressionAttributeValues=None,
+        UpdateExpression=None,
+    ):
+        pass
```

Comment:
Why isn't there a test for this method?

Reply:
The diff makes this look weird. This is a function for mock `DynamoDbMockPutItem` so it doesn't need a test